www/app/common/update-eula.service.js

```diff
@@ -45,8 +45,8 @@
           },
         })
         .catch(function() {
-          console.log('error submitting eula');
+          console.log('Error submitting EULA!');
         });
     }
   }
-})();
```

Comment:
Github did it. It's good practice to have an empty line at EOF anyway! http://stackoverflow.com/a/2287990